stdlib/public/stubs/Stubs.cpp

```diff
@@ -64,6 +64,10 @@ static long double swift_strtold_l(const char *nptr,
 #define strtod_l swift_strtod_l
 #define strtof_l swift_strtof_l
 #define strtold_l swift_strtold_l
+#elif defined(__Fuchsia__)
+// this might be for all MUSL libc implementations
```

Comment:
Fuchsia's libc has already diverged sufficiently from musl to the point where they're not interchangeable, this came up in libc++ where we originally just used musl support but since then we introduced Fuchsia support because of the differences.